lib/install/validate/validator.go

```diff
@@ -277,6 +277,10 @@ func (v *Validator) network(ctx context.Context, input *data.Data, conf *metadat
 			Default: true, // external network is default for appliance
 		},
 	})
+	// Bridge network should be different with all other networks
+	if input.BridgeNetworkName == input.ExternalNetworkName {
+		v.NoteIssue(errors.Errorf("bridge network %s is used for external network, please choose different network", input.BridgeNetworkName))
```

Comment:
the bridge network must not be shared with another network role - external also uses %s